doc/sample-nginx.conf

```diff
@@ -0,0 +1,62 @@
+worker_processes  1;
+
+events {
+    worker_connections 1024;
+}
+
+http {
+    # Blockdaemon with bearer token authentication
+    server {
+        listen 7900;
+
+        location / {
+            proxy_pass https://solana-01.bdnodes.net;
+            proxy_http_version 1.1;
+            proxy_set_header Host $proxy_host;
+            proxy_set_header Authorization "Bearer ReplaceMeWithYourAuthToken";
+            proxy_redirect https://$proxy_host http://$server_name:$server_port;
+            proxy_redirect https:// http://;
+        }
+    }
+
+    # Figment RPC with auth
+    server {
+        listen 7901;
+
+        location / {
+            proxy_pass https://solana--mainnet--rpc.datahub.figment.io/apikey/ReplaceMeWithYourAuthToken;
+            proxy_http_version 1.1;
+            proxy_set_header Host $proxy_host;
+            proxy_redirect https://$proxy_host http://$server_name:$server_port;
+            proxy_redirect https:// http://;
+        }
+    }
+
+    # Figment websocket with auth
+    server {
+        listen 7902;
+
+        location / {
+            proxy_pass https://solana--mainnet--ws.datahub.figment.io/apikey/ReplaceMeWithYourAuthToken;
+            proxy_http_version 1.1;
+            proxy_set_header Host $proxy_host;
+            proxy_set_header Upgrade $http_upgrade;
+            proxy_set_header Connection "Upgrade";
+            proxy_redirect https://$proxy_host http://$server_name:$server_port;
+            proxy_redirect https:// http://;
+        }
+    }
+
+    # TritonOne RPC without explicit auth
+    server {
+        listen 7903;
+
+        location / {
+            proxy_pass https://yourserver.rpcpool.com;
+            proxy_http_version 1.1;
+            proxy_set_header Host $proxy_host;
```

Comment:
Not sure if this is strictly required - nginx should do this automatically

Reply:
Shouldn't be required but deliberately being explicit since this is intended as instructional samples for publishers to use.